lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -335,6 +335,12 @@ qx.Class.define("callbackery.ui.form.Auto", {
             this._settingData = false;
         },
 
+        setSelectBoxSelection: function(box, selection) {
+            this._settingData = true;
+            var ctrl = this._boxCtrl[box];
+            ctrl.setSelection(new qx.data.Array([selection]));
+            this._settingData = false;
```

Comment:
```suggestion
            var ctrl = this._boxCtrl[box];
            ctrl.setSelection(new qx.data.Array([selection]));
```

setting data is not necessary as this is a synchronous call